### DISCUSSION TITLE
make publish branch with only the latest commit

### DISCUSSION THREAD

**Discussion by liblaf** [upptime/upptime#713] [Ideas]
The history of deployments in the `gh-pages` branch is not very important. Maybe we can save storage usage by setting the `force_orphan` option of `peaceiris/actions-gh-pages` to `true` to only keep the latest deployment.

**Comment by AnandChowdhary** [comment]
Agreed! Happy to accept a PR :)

**Reply by liblaf** [reply]
I have made a simple PR at <https://github.com/upptime/uptime-monitor/pull/208>. Please let me know if the PR does not meet the requirements.
